fypy/pricing/analytical/spread_option_kirk_method.py

```diff
@@ -0,0 +1,38 @@
+"""
+About: Pricing Spread option using Kirk 2D method
+"""
+
+import math
+import scipy.stats as stats
+
+def spread_option_kirk_method(K : float,
+                              S0_1 : float,
+                              S0_2 : float,
+                              T : float,
+                              r: float,
+                              rho: float,
+                              sigma_1: float,
+                              sigma_2: float,
+                              q_1: float,
+                              q_2: float) -> float:
+    """
+    Params:
+    K: Option Strike
```

Comment:
@jkirkby3  do we expect param commenting like
_:param K: Option strike
:param S0_1: Initial Price of asset 1_
etc.?

Reply:
Yea nice catch @nrupprecht ... @abhinav1630 , can you update the documentation style to match the rest of library? You can configure this in your IDE.

Reply:
Thanks @nrupprecht. Have updated that